torchvision/prototype/transforms/_color.py

```diff
@@ -147,7 +147,7 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
             inpt = F.adjust_contrast(
                 inpt, contrast_factor=ColorJitter._generate_value(self.contrast[0], self.contrast[1])
             )
-        if params["channel_permutation"]:
+        if params["channel_permutation"] is not None:
```

Comment:
This is an actual bug. While `if x` works correctly for `None`, i.e. no channel permutation, it fails if we actually have one:

```py
>>> bool(None)
False
>>> bool(torch.tensor([2, 0, 1]))
RuntimeError: Boolean value of Tensor with more than one value is ambiguous
```